README.md

```diff
@@ -15,7 +15,8 @@ In order to create a trigger that reacts when messages are posted to a Message H
 |password|String|Your Message Hub password|
 |topic|String|The topic you would like the trigger to listen to|
 |kafka_admin_url|URL String|The URL of the Message Hub admin REST interface|
-|isJSONData|Boolean (Optional - default=false)|When set to `true` this will cause the feed to attempt the message content as JSON before passing it along as the trigger payload.|
+|isJSONData|Boolean (Optional - default=false)|When set to `true` this will cause the provider to attempt to parse the message content as JSON before passing it along as the trigger payload.|
+|isBinaryData|Boolean (Optional - default=false)|When set to `true` this will cause the provider to attempt to encode the message content as Base64 before passing it along as the trigger payload.|
```

Comment:
Strictly speaking, I am not sure base64 encoding can ever fail.. if this is indeed the case, maybe drop the 'attempt' part..